demisto_sdk/commands/generate_docs/tests/generate_docs_test.py

```diff
@@ -584,6 +584,32 @@ def test_generate_integration_doc(self):
         assert "Number of users to return. Max 300. Default is 30." in open(fake_readme).read()
 
 
+def test_get_command_examples(tmp_path):
+    """
+        Given
+            - command_examples file.
+            - list of specific commands
+        When
+            - Running get_command_examples with the given command examples and specific commands.
+        Then
+            - Verify that the returned commands from the examples are only the specific sommands
+    """
+    command_examples = tmp_path / "command_examples"
+    command_examples.touch()
+
+    with open(command_examples, 'w') as ce:
```

Comment:
You can use `w+` flag so you won't need the `command_examples.touch()`. But it's just my favor, if you want.

Reply:
done.